<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureMIDILine</class>
 <widget class="QDialog" name="ConfigureMIDILine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure MIDI Input Line</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="m_feedBackLabel">
     <property name="text">
      <string>Feedback Line</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="m_feedBackCombo">
     <property name="toolTip">
      <string>The MIDI line that is used to send feedback data to the input device (to move motorized sliders etc.)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="m_midiChannelLabel">
     <property name="text">
      <string>MIDI Channel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="m_midiChannelCombo">
     <item>
      <property name="text">
       <string>Channel 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Any Channel</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureMIDILine</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureMIDILine</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
